.github/workflows/dev_container.yml

```diff
@@ -0,0 +1,39 @@
+
+name: Container build
+
+on:
+  push:
+    branches:
+      - 'main'
+      - 'stable'
+      - 'beta'
+      - 'release/**'
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    name: Build Container
+    runs-on: [runs-on,runner=8cpu-linux-x64,"image=ubuntu24-full-x64","run-id=${{ github.run_id }}"]
+    steps:
+
+      - uses: actions/checkout@v4
+
+      - name: Build and load container image
+        uses: docker/build-push-action@v6
+        with:
+          push: false
+          load: true
+          tags: ghcr.io/px4/px4-dev:latest
```

Comment:
Is this right? Generally `latest` isn't a tag you explicit create.

Reply:
yes we need to lock down the tag, for now I'm using latest until I can get a clean build going